.github/workflows/ci-pipeline.yaml

```diff
@@ -89,14 +90,16 @@ jobs:
     needs: get-chart
     # Given performance issues of the action feature on GH's side, we need to be very restrictive in the job's triggers:
     # -> The 'Get modified charts' job suceededs AND
-    # -> The event action is different from opened AND
-    #  ( ---> The pipeline was triggered due to a label addition and said label was the 'verify' one OR
-    #    ---> The pipeline was NOT triggered due to a label addition but the PR already contains the 'verify' one )
+    # -> It isn't an opened action (it avoids very running twice on consecutive opened and labeled events)  AND
+    #  ( ---> The pipeline was triggered due to a label addition and said label was the 'verify' one) OR
+    #  ( ---> The pipeline was NOT triggered due to a label addition but the PR already contains the 'verify' one )
```

Comment:
Either add extra surrounding parenthesis or delete the added ones because

```
a AND b AND (c AND d) OR (e AND f)
```

is not the intended logic:

```
a AND b AND ((c AND d) OR (e AND f))
```